<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsHistoryPage</class>
 <widget class="QWidget" name="ResultsHistoryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>688</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,12,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,15,2">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="history_results_title_">
        <property name="text">
         <string>MLPerf Client History Results </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="history_results_container_" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>25</number>
           </property>
           <property name="topMargin">
            <number>12</number>
           </property>
           <property name="rightMargin">
            <number>25</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <item>
            <widget class="FilterButton" name="sort_by_button_">
             <property name="minimumSize">
              <size>
               <width>156</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>156</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string>Sort by</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="FilterButton" name="device_type_button_">
             <property name="minimumSize">
              <size>
               <width>156</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>156</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string>Device type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="select_all_button_">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Select All</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deselect_all_button_">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Deselect All</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="delete_button_">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>40</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Delete  </string>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/icons/resources/icons/delete.png</normalon>
               <disabledon>:/icons/resources/icons/delete_disabled.png</disabledon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="open_report_button_">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>168</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Compare</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListView" name="history_list_view_"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterButton</class>
   <extends>QPushButton</extends>
   <header>ui/widgets/filter_button.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../app_resources.qrc"/>
 </resources>
 <connections/>
</ui>
